www-static/client/world.js

```diff
@@ -410,7 +410,8 @@ CCC.World.prerenderPanorama = function(node) {
 CCC.World.connectPanel = function(node) {
   var df = document.createDocumentFragment();
   var div = document.createElement('div');
-  div.className = 'systemDiv';
+  div.className =
+      (CCC.Common.isConnected ? 'connectDiv' : 'disconnectDiv');
   var msg = CCC.World.getMsg(node.isConnected ?
```

Comment:
Why is the line above conditional on `ccc.Common.isConnected` but this one on `node.isConnected`?  That smells bad (or at least like it deserves a comment).